musicalbum.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+
+class Musicalbum < Item
+  attr_reader :id
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, publish_date, archived: false, id: nil)
+    super(id, publish_date, archived: archived)
+    @on_spotify = on_spotify
```

Comment:
Hi @Hisoka37 
Your work is complete 💯 
no linter errors 👍 
![RecursiveThumbsUpGIF](https://github.com/Microverse-Fullstack-Program/catalog-of-things/assets/54628727/a94c3107-5bae-425c-bd26-a211e0c60d9c)